src/lib.rs

```diff
@@ -243,26 +243,6 @@ macro_rules! mask {
 }
 
 impl CidrSubnet {
-	/// Obtains the CidrSubnet from the well-known
-	/// addr/prefix notation.
-	/// ```
-	/// # use std::str::FromStr;
-	/// # use rcgen::CidrSubnet;
-	/// // The "192.0.2.0/24" example from
-	/// // https://tools.ietf.org/html/rfc5280#page-42
-	/// let subnet = CidrSubnet::from_str("192.0.2.0/24").unwrap();
-	/// assert_eq!(subnet, CidrSubnet::V4([0xC0, 0x00, 0x02, 0x00], [0xFF, 0xFF, 0xFF, 0x00]));
-	/// ```
-	pub fn from_str(s :&str) -> Result<Self, ()> {
```

Comment:
This is an API breaking change. I also wonder how implementing this trait benefits users. In fact usage is harder now as you have to import the `FromStr` trait. It's also way more hidden in the rustdoc. So please don't do it.

Reply:
I think using traits benefits almost everyone. The auto-generated documentation for this crate will still mention `from_str`.